main.js

```diff
@@ -0,0 +1,231 @@
+    url: "https://api.openweathermap.org/data/2.5/weather?q=" + query + "&appid=5e5a03c24f89b112d6b16ccfccf14576",
+    dataType: "json",
+    success: function (data) {
+      
+      addWeather(data);
+
+    },
+    error: function (jqXHR, textStatus, errorThrown) {
+      debugger;
+      console.log(textStatus);
+    }
+  });
+};
+
+
+$('.search').on('click', function () {
+  var search = $('#search-query').val();
+  
+  fetch(search);
+  fetchFiveDay(search);
+  
+});
+
+var addWeather = function (data) {
+  
+  currentWeatherArray = [];
+  
+  var weather = {
+    name: data.name || null,
+    currentTemp: Math.floor(kelvinToF(data.main.temp)) + '°' || null,
+    main: data.weather[0].main || null,
+    icon: 'http://openweathermap.org/img/wn/' + data.weather[0].icon +'@2x.png'|| null
+  };  
+  currentWeatherArray.push(weather);
+    renderCurrentWeather();
+    
+  };
+
+
+var renderCurrentWeather = function() {
+  $('.current-forecast').empty();
+
+  for (let i = 0; i < currentWeatherArray.length; i++) {
+    const weather = currentWeatherArray[i];
+
+    var source = $('#current-weather-template').html();
+    var template = Handlebars.compile(source);
+    var newHTML = template(weather);
+   
+
+    $('.current-forecast').append(newHTML);
+
+    
+  }
+};
+
+
+renderCurrentWeather();
+
+
+function kelvinToF (number) {
+  return (number - 273.15) * 9/5 + 32;
+};
+
+
+function convertDate (date) {
+
+  var dateToDay = new Date(date);
+  var dayToNumber = dateToDay.getDay();
+
+  const daysOfTheWeek = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
+
+  var day = (daysOfTheWeek[dayToNumber]);
+  return day;
+}
+
+
+var fiveDayWeather = [
+  {
+    mainWeather: null,
+    day: null,
+    temp: null,
+    icon: null
+  },
+  {
+    mainWeather: null,
+    day: null,
+    temp: null,
+    icon: null
+  },
+  {
+    mainWeather: null,
+    day: null,
+    temp: null,
+    icon: null
+  },
+  {
+    mainWeather: null,
+    day: null,
+    temp: null,
+    icon: null
+  },
+  {
+    mainWeather: null,
+    day: null,
+    temp: null,
+    icon: null
+  }
+];
+
+
+var addFiveDayWeather = {
+
+  day1Average: function(data) {
+    var day1Sum = 0;
+    var day1Average = 0;
+      for (let i = 0; i <= 7; i++) {
+      day1Sum = data.list[i].main.temp + day1Sum;
+      };
+    day1Average = Math.floor(kelvinToF(day1Sum / 8));
+    fiveDayWeather[0].temp = (day1Average + '°');
+    
+    fiveDayWeather[0].mainWeather = (data.list[7].weather[0].main);
+    fiveDayWeather[0].icon = 'http://openweathermap.org/img/wn/' + data.list[7].weather[0].icon +'@2x.png';
+    fiveDayWeather[0].day = convertDate(data.list[7].dt_txt);
+  },
+
+  day2Average: function(data) {
+    var day2Sum = 0;
+    var day2Average = 0;
+      for (let i = 8; i <= 15; i++) {
+      day2Sum = data.list[i].main.temp + day2Sum;
+      };
+    day2Average = Math.floor(kelvinToF(day2Sum / 8));
+    fiveDayWeather[1].temp = (day2Average  + '°');
+
```

Comment:
This works but isn't very DRY (don't repeat yourself) - could you accomplish this in a more readable/efficient way?